akka-docs/src/test/java/jdocs/actor/FaultHandlingDocSample.java

```diff
@@ -67,7 +67,7 @@ public static void main(String[] args) {
     public void preStart() {
       // If we don't get any progress within 15 seconds then the service
       // is unavailable
-      getContext().setReceiveTimeout(Duration.create("15 seconds"));
+      getContext().setReceiveTimeout(java.time.Duration.ofSeconds(15));
```

Comment:
Same as the other one, make this the default import and use a fqn for the Scala one if it is used

Reply:
Sorry, i didn't see that the last time.